### DISCUSSION TITLE
posts/yocto-project-on-orange-pi-2/

### DISCUSSION THREAD

**Discussion by giscus** [humminglab/blog.humminglab.io#6] [Announcements]
# posts/yocto-project-on-orange-pi-2/

이전 글 에서 meta-sunxi 를 추가하여 orage pi 용으로 빌드를 만들었고, 이번 과정은 project 용으로 meta layer를 만들어서 관리하는 방법을 설명한다.
실제 개발 과정을 이해하기 좋도록 meta layer 를 만들어 가는 과정을 설명한다.
Yocto Project 개발하기(1) - Orange Pi 보드 빌드 Yocto Project 개발하기(2) - Custom Layer 만들기 Yocto Project 개발하기(3) - 개발 시 로컬 패키지 관리하기 Yocto Project 개발하기(4) - Yocto SDK 빌드 Yocto Project 개발하기(5) - Yocto eSDK 이용한 개발 모델 Layer 및 machine 생성 프로젝트를 sc-gateway 라고 명하고(이름에 특별한 의미는 없음), 이를 layer로 만든다.

https://blog.humminglab.io/posts/yocto-project-on-orange-pi-2/

**Comment by kdj6724** [comment]
yoctoproject.org에서 며칠을 헤멨던 걸 단 두페이지로 정리해주시다니 좋은 설명 감사합니다.

**Reply by aqwerf** [reply]
도움이 되셨다니 저도 기분이 좋네요^^ 
프로젝트 규모가 크지 않으면 yocto가 큰 장점을 발휘하지 못하여서인지, 국내에서는 많이 사용하지는 않는 것 같아요. 

참고로 Yocto 에 OSTree를 적용한 것도 유사한 시리즈니 한번 보시는 것도 좋을 것 같습니다. 

- [Yocto에 OSTree upgrade 적용(1) - 이미지 생성](https://blog.humminglab.io/posts/yocto-ostree-meta-updater-1/)
- [Yocto에 OSTree upgrade 적용(2) - 부팅 절차](https://blog.humminglab.io/posts/yocto-ostree-meta-updater-2/)
- [Yocto에 OSTree upgrade 적용(3) - 업그레이드/롤백 및 OSTree 리뷰](https://blog.humminglab.io/posts/yocto-ostree-meta-updater-3/)

**Comment by keikkang** [comment]
레이어 관련 최고의 레퍼런스

**Reply by aqwerf** [reply]
감사합니다. 
잊고 있었는데, 올해는 yocto 에 대해서 좀 더 보완을 해야 겠네요.
